### DISCUSSION TITLE
Suitable uses?

### DISCUSSION THREAD

**Discussion by janeblonde** [Attendize/Attendize#1031] [Q&A] [answered]
HI Guys,

I'm looking for a self hosted ticket selling platform that's reliable and can work with multiple venues etc.

The basic requirements we have are simple, sell digital tickets that can be printed and scanned. We will need an app so scanning can be done on a phone a the venue doors.

I keep coming across Attendize and would like to try it but I'm worried that development is stalling as the website is down and runs off php 7.1.x.

What do you think, am I wrong to be worried or I should just install and take it from there?

DDH

**Comment by johannac** [comment, marked as answer]
Attendize does cover the use cases you listed in your question, and it's free to try it out... so I'd say maybe give it a try! 😃  Also the web site is back up now, thanks for letting us know! Been having a ton of problems with the DNS and github pages lately.. hopefully it's all sorted now..
